server/db/order.js

```diff
@@ -0,0 +1,21 @@
+const Sequelize = require ('sequelize');
+const db = require('./_db');
+//import join here
+
+const Order = db.define('order', {
+  date: {
+    type: Sequelize.DATEONLY
+  }, 
+  totalCost: {
+    type: Sequelize.INTEGER
+  }, 
+  status: {
+    type: Sequelize.ENUM,
+    values: ['pending', 'completed']
+  }, 
+  totalQuantity: {
```

Comment:
same as TOTAL, this is another attribute we can pull by querying the join table

Reply:
Done!